DEVELOPMENT_WORKFLOW.md

```diff
@@ -2,97 +2,97 @@
-    1. [Git operations](#git-operations)
-        1. [Create a feature branch](#create-a-feature-branch)
-        1. [Merge feature branch into nobunaga](#merge-feature-branch-into-nobunaga)
-        1. [Create release branch](#create-release-branch)
-        1. [Finish a release branch](#finish-a-release-branch)
-        1. [Create a hotfix branch](#create-a-hotfix-branch)
-        1. [Finishing a hotfix branch](#finishing-a-hotfix-branch)
+- [Development Workflow](#development-workflow)
+  - [Table of contents](#table-of-contents)
+  - [Main branches](#main-branches)
+  - [Development model](#development-model)
+      - [Feature branches](#feature-branches)
+      - [Release branches](#release-branches)
+      - [Hotfix branches](#hotfix-branches)
+    - [Git operations](#git-operations)
+      - [Create a feature branch](#create-a-feature-branch)
+      - [Merge feature branch into staging](#merge-feature-branch-into-staging)
+      - [Create release branch](#create-release-branch)
+      - [Finish a release branch](#finish-a-release-branch)
+      - [Create the hotfix branch](#create-the-hotfix-branch)
+      - [Finishing a hotfix branch](#finishing-a-hotfix-branch)
+  - [TODO](#todo)
 
 ## Main branches
 
-The repo holds two main branches with an infinite lifetime:
-- master
-- nobunaga
+Bittensor is composed of TWO main branches, **master** and **staging**
 
-We consider `origin/master` to be the main branch where the source code of HEAD always reflects a **__production-ready__** state.
+**master**
+- master Bittensor's live production branch. This branch should only be touched and merged into by the core develpment team. This branch is protected, but you should make no attempt to push or merge into it reguardless. 
 
-We consider `origin/nobunaga` to be the main branch where the source code of HEAD always reflects a state with the **__latest delivered development__** changes for the next release. Some would call this the `"integration branch"`. This is where any automatic nightly builds would be built from.
+**staging**
 
-## Development model
-
-### Supporting branches
+- staging is Bittensor's development branch. This branch is being continuously updated and merged into. This is the branch where you will propose and merge changes.
 
-Each of these branches have a specific purpose and are bound to strict rules as to which branches may be their originating branch and which branches must be their merge targets. We will walk through them in a minute
+## Development model
 
 #### Feature branches
 
-- May branch off from: `nobunaga`
-- Must merge back into: `nobunaga`
-- Branch naming convention:
-    - Anything except master, nobunaga, nakamoto, release/* or hotfix/*
-    - Suggested: `feature/<ticket>/<descriptive-sentence>`
+    When implementing new features, hotfixes, bugfixes, or upgrades, you must adhere to a STRICT naming and merging convention.
+
+**Branch naming and merging convention:**
 
+  
```

Comment:
i'm missing naming convention and intructions for feature branches. We should expose it as we have for release branches